fission-web-server/library/Fission/Web/Server/Handler/Append.hs

```diff
@@ -36,6 +35,6 @@ handlerV2 ::
   => Append.RoutesV2 (AsServerT m)
 handlerV2 = Append.RoutesV2 {append}
   where
-    append appName fileName (Serialized rawData) Authorization {about = Entity userId _} = do
-      cid <- Web.Err.ensureM $ App.addFile userId appName fileName rawData
+    append appName fileName (Serialized rawData) Authorization {about = Entity userId _, potency, resource} = do
```

Comment:
Does this `userId` represent the root issuer in the UCAN? From testing, it seems like it does, but it's not clear to me where we are pulling the UCAN apart to get that.

Reply:
It does. That happens here: https://github.com/fission-suite/fission/blob/fa3d2b52848fe3573714a9a6c3611555461c13bc/fission-web-server/library/Fission/Web/Server/Auth/Token/UCAN.hs#L55

Reply:
Ah great! Thanks for the reference.